src/Widget/WidgetPageInterface.php

```diff
@@ -170,6 +170,20 @@ public function setCss($css);
      */
     public function getCss();
 
+    /**
+     * Set the selected theme
+     *
+     * @param string $selectedTheme
+     */
+    public function setSelectedTheme($selectedTheme);
```

Comment:
If it's not intentionally dynamic, I would suggest painting the argument to `string`